sos/report/plugins/filesys.py

```diff
@@ -108,6 +108,6 @@ class RedHatFilesys(Filesys, RedHatPlugin):
 
     def setup(self):
         super().setup()
-        self.add_cmd_output("ls -ltradZ /tmp")
+        self.add_dir_listing('/tmp')
```

Comment:
what this did before, and what it will give now are 2 distinct things, not sure how we best want to do that

Reply:
Ack, will move this one back to `ls` directly. `a` also is meaningless with `d`, so this will become `ls -ldZ /tmp`.